samconfig.toml

```diff
@@ -2,11 +2,11 @@ version = 0.1
 [default]
 [default.deploy]
 [default.deploy.parameters]
-stack_name = "karate-test-cristin-service"
+stack_name = "np3392-karate-cristin-service"
```

Comment:
Not sure if you want to commit this file…?

Reply:
As we have discussed in chat, I have to do some changes to this file, maybe reverting it before merge. We have to come up with something